### PR TITLE
App privacy manifest file

### PR DESCRIPTION
**Pull request by jsonDaCruzo**
Apple's new privacy policy requests apps to specify for any SDKs used the types of data collected and the corresponding purpose. This file lists this specification for the Qubit iOS SDK.

### CODE REVIEW

**Review by ravi-qubit — APPROVED** [review]
Looks good, one comment to remove empty element in `NSPrivacyAccessedAPITypeReasons` within PrivacyInfo file.

File: QBTracker/QubitSDK/PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		</dict>
+	</array>
+	<key>NSPrivacyTrackingDomains</key>
+	<array/>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyAccessedAPITypeReasons</key>
+			<array>
+				<string></string>
```
[the remark this answers is not in the document]
Comment (jsonDaCruzo) [reply QBTracker/QubitSDK/PrivacyInfo.xcprivacy:79]: @mbalawajder do you agree with us removing this empty string?

[the remark this answers is not in the document]
Comment (mbalawajdermq) [reply QBTracker/QubitSDK/PrivacyInfo.xcprivacy]: @jsonDaCruzo done

[the remark this answers is not in the document]
Comment (jsonDaCruzo) [reply QBTracker/QubitSDK/PrivacyInfo.xcprivacy]: Thanks!

**Review by hporter — APPROVED** [review]

**Pull request merged**